frontend/src/components/ui/ClientStationFilters.tsx

```diff
@@ -58,6 +60,11 @@ const ClientStationFilters = () => {
             {filterOptions[filterKey].label}
           </ClientFilterButton>
         ))}
+        {process.env.NODE_ENV === 'development' && (
+          <ClientFilterButton $isChecked={false}>
+            Zoom: {zoom.zoom} ({zoom.state})
+          </ClientFilterButton>
+        )}
```

Comment:
```suggestion
        // TODO: 추후 삭제될 로직입니다
        {process.env.NODE_ENV === 'development' && (
          <ClientFilterButton $isChecked={false}>
            Zoom: {zoom.zoom} ({zoom.state})
          </ClientFilterButton>
        )}
```

이런 주석이 포함되면 좋을 것 같네요!